UI/GameSettingsScreen.cpp

```diff
@@ -350,13 +350,14 @@ void GameSettingsScreen::CreateViews() {
 	}
 #endif
 
-#ifdef _WIN32
+#if !PPSSPP_PLATFORM(ANDROID)
```

Comment:
It's Vulkan only though, right?  Worried this may cause confusion for people on OpenGL, i.e. Raspberry pi, etc.

-[Unknown]

Reply:
Fully aware of this - which I will fix in a followup by simply adding support for toggling VSync on these platforms too.